### PR TITLE
Fixed actual_main to correctly return 0 on success and made

### PR DESCRIPTION
**Pull request by PhantomHoover**
-shadow_quality imply -enable_shadows.

### CODE REVIEW

**Comment by chief1983** [comment]
Won't changing the game's return require changing all the launchers that detect the exit code?

**Comment by PhantomHoover** [comment]
Are there launchers that do that?

**Comment by chief1983** [comment]
I was pretty sure wxLauncher monitors the running game and checks the exit code when the game exits.

**Comment by PhantomHoover** [comment]
Oh yeah, it does, but the only thing it actually does with the code is report that "FS2 Open exited with a status of %d".

**Comment by MageKing17** [comment]
The `-shadow_quality` change would be better- accomplished by simply moving the inner conditional to be outside the `-enable_shadows` conditional.

**Comment by PhantomHoover** [comment]
That would be semantically different, in that it would make -enable_shadows have no effect whatsoever.

**Comment by MageKing17** [comment]
What? No it wouldn't; it would set shadow quality to 2, just as it currently does.

**Comment by PhantomHoover** [comment]
Oh right you mean putting it in a parallel block. Good point.

File: freespace2/freespace.cpp [hunk 1]
```diff
@@ -6920,8 +6920,8 @@ DCF(pofspew, "Spews POF info without shutting down the game")
 }
 
 // returns:
```
Comment (z64555) [inline freespace2/freespace.cpp]: Could you write this into doxygen's format? ex:

`/**
 * @brief description here
 *
 * @returns 0 on a clean exit, or
 * @returns 1 on an error
 */`

**Pull request merged**